spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,16 +37,24 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile compatible implementation must pass *all* TCKs provided by each MicroProfile specification. Passing the
```

Comment:
Maybe also mention MP TCKs that are optional? If we have something like this in some of the platform specs. Not sure, I would need to check.

Reply:
Ah, you mean the optional tests for EJB integration in JWT for instance?

Reply:
@radcortez exactly. Also, there might be more optional TCKs in the future.